README.md

```diff
@@ -74,15 +81,22 @@ Finally, [eoapi-template](https://github.com/developmentseed/eoapi-template) is
 
 ## Contribution & Development
 
-See [CONTRIBUTING.md](https://github.com/developmentseed/eoAPI/blob/main/CONTRIBUTING.md)
+We highly value and rely on our community! Whether you're an expert or just getting started, you can make a difference. Here's how:
+
+- **Contribute**: Check out our [CONTRIBUTING.md](https://github.com/developmentseed/eoAPI/blob/main/CONTRIBUTING.md) guide to understand how you can contribute.
+- **Engage in Discussions**: Share your ideas, ask questions, or provide feedback through [GitHub Discussions](https://github.com/developmentseed/eoAPI/discussions). This is where most of our project conversations take place.
+- **Report Issues**: Found a bug or have a feature request? Raise it on our [issues page](https://github.com/developmentseed/eoAPI/issues).
 
+---
 ## License
 
-See [LICENSE](https://github.com/developmentseed/eoAPI/blob/main/LICENSE)
+At Development Seed, we believe in open collaboration and making tools and data more accessible. In line with this ethos, we've explicitly chosen a less restrictive license for `eoAPI`.
```

Comment:
You can name the license here, don't make people click to find out it's MIT licensed, but do keep the link.